lang/lang.go

```diff
@@ -203,7 +203,5 @@ func updateLocalizer() {
 		fyne.LogError("Failed to load user locales", err)
 		all = []string{"en"}
 	}
-	str := closestSupportedLocale(all).LanguageString()
-	setupLang(str)
-	localizer = i18n.NewLocalizer(bundle, str)
```

Comment:
Are you sure this was superfluous?
The intention was that it would replace whatever previous configuration was set up...

Reply:
`setupLang` does the exact same thing, see https://github.com/fyne-io/fyne/blob/develop/lang/lang.go#L196

Reply:
Excellent, sorry for missing that - I was on my mobile